pkg/view/component/donut_chart.go

```diff
@@ -63,6 +67,10 @@ func (dc *DonutChart) SetSize(size DonutChartSize) {
 	dc.Config.Size = size
 }
 
+func (dc *DonutChart) SetThickness(thickness int) {
+	dc.Config.Thickness = thickness
```

Comment:
As a plugin developer, how will I know what is a reasonable number for thickness?

Negative thickness also makes interesting charts, but maybe it shouldn't be allowed
![image](https://user-images.githubusercontent.com/10288252/105777517-5d5e0600-5f1f-11eb-8ec0-e3e1b7b63c17.png)

Reply:
Right. Trimming this to [2-100] now. Also added comment to `SetThickness` method and story.